portfolio/src/main/webapp/style.css

```diff
@@ -1,6 +1,7 @@
 body {
-  background: url("/images/moroccan-flower-dark.png"), #6F4A38;
+  background: #00274C;
```

Comment:
to lower cases #00274c

https://google.github.io/styleguide/htmlcssguide.html#Capitalization